### PR TITLE
fix: various auctions UI fixes

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Bugfix**

### Description
This PR resolves layout issues reported by sam here [MX-515](https://artsyproduct.atlassian.net/browse/MX-515), here [MX-517](https://artsyproduct.atlassian.net/browse/MX-517) and here [MX-519](https://artsyproduct.atlassian.net/browse/MX-519)

**Hide Patner Name**
Screenshot 2020-09-29 at 13 06 25

**Remove Upper case from lot label**
Screenshot 2020-09-29 at 13 06 03

**Dynamically adjust aspect ratio**
Screenshot 2020-09-29 at 13 05 43

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))

#trivial

### CODE REVIEW

File: src/lib/Scenes/Sale/Components/SaleArtworkListItem.tsx [hunk 1]
```diff
@@ -55,15 +65,25 @@ export const SaleArtworkListItem: React.FC<Props> = ({ artwork }) => {
   )
 }
 
-const styles = StyleSheet.create({
-  artworkImage: {
-    justifyContent: "flex-end",
-    paddingHorizontal: 5,
-    paddingBottom: 5,
+// Get image accurate square dimensions while keeping the same aspect ratio
```
Comment (MounirDhahri) [inline src/lib/Scenes/Sale/Components/SaleArtworkListItem.tsx:68]: we might need to export this to a helper in the future

**Comment by admbtlr** [comment]
Maybe change the PR title to `fix: various auctions UI fixes`? 😬

**Comment by MounirDhahri** [comment]
Yes sure. I was assuming whoever squashes and merges will set the commit title

**Comment by admbtlr** [comment]
I think it's best for the author to do it, since they are more invested in the PR. Plus that adds another level of checking: if the author forgets, then hopefully the merger will remember - but more eyes are better!

**Comment by MounirDhahri** [comment]
You're right. Thanks for explaining 🙏

**Review by pvinis — APPROVED** [review]

**Pull request merged**